Dockerfile

```diff
@@ -1,12 +1,17 @@
 FROM alpine:latest
-RUN apk update
-RUN apk add build-base python3-dev py3-pip jpeg-dev zlib-dev shadow git
-WORKDIR /cameraobscura  
-RUN git clone https://github.com/roastingmalware/cameraobscura.git .
-RUN LIBRARY_PATH=/lib:/usr/lib /bin/sh -c "python3 -m pip install -r requirements.txt"
+
+WORKDIR /cameraobscura
+
+COPY . .
+
+RUN apk update; \
+    apk add build-base python3-dev py3-pip jpeg-dev zlib-dev shadow; \
+    LIBRARY_PATH=/lib:/usr/lib /bin/sh -c "python3 -m pip install -r requirements.txt"; \
+    groupadd -r obscura && useradd -r -g obscura -d /cameraobscura -s /sbin/nologin -c "Docker image user" obscura; \
+    chown obscura:obscura /cameraobscura -R;
+
 EXPOSE 8080
-RUN groupadd -r obscura && useradd -r -g obscura -d /cameraobscura -s /sbin/nologin -c "Docker image user" obscura
-RUN chmod +x obscura.sh
-RUN chown obscura:obscura /cameraobscura -R
 USER obscura
-CMD python3 main.py
+
+ENTRYPOINT [ "python3" ]
```

Comment:
Nitpick: I know python3 is in the path, but it might be more robust, to set it to its full path here (just checked against a current alpine: `/usr/bin/python3`), correct? That eliminates one more source of errors. What do you think?
Also: This combination of ENTRYPOINT/CMD is a great idea, thanks!

Reply:
I agree, it could be more explicit. Or maybe even better solution would be to switch to `FROM python:3-alpine` image, with python already installed.

Reply:
Let's have a look at that together. I haven't seem the project in quite some time and I am unsure what happens if we switch the base image. @cmllr do you see any problems there?